scanner/src/main/kotlin/storages/PostgresStorage.kt

```diff
@@ -101,6 +99,8 @@ class PostgresStorage(private val connection: Connection) : ScanResultsStorage()
     }
 
     override fun readFromStorage(id: Identifier): ScanResultContainer {
+        log.info { "Reading scan results for ${id.toCoordinates()} from storage." }
```

Comment:
Should these log statements already be introduced in the previous commit?

Reply:
Done.